tests/integration/web/devicehistory_test.py

```diff
@@ -151,7 +156,8 @@ def test_get_module_history_should_succeed(db, client, localhost):
 
     url = reverse('devicehistory-view')
     response = client.get(
-        f"{url}?from_date=2023-01-01&to_date=2025-01-01&eventtype=all&module={module.id}&submit_module=View+romoduleom+history"
+        f"{url}?from_date=2023-01-01&to_date=2025-01-01&eventtype=all"
+        + f"&module={module.id}&submit_module=View+romoduleom+history"
```

Comment:
Did you use a tool to break these up, or did you do it manually?  I don't see the need for this redundant `+` operator within parentheses - it's not even consistent throughout :)

Reply:
(this usage of `+` appears in more places than just the one I commented)